Resources/public/js/views/fields/ez-relationlist-editview.js

```diff
@@ -50,7 +52,57 @@ YUI.add('ez-relationlist-editview', function (Y) {
                 } else {
                     this.render();
                 }
+
+                clearTimeout(this.makeListSortableTimeout);
+                this.makeListSortableTimeout = setTimeout(Y.bind(this._makeListSortable, this), 75);
+            });
+        },
+        
+        /**
+         * Sets up the relation list to be sortable
+         */
+        _makeListSortable: function() {
+            var fieldDefinitionIdentifier = this.getField().fieldDefinitionIdentifier;
+    
+            var sortable = new Y.Sortable({
+                container: "." + fieldDefinitionIdentifier + ".ez-relation-container",
+                nodes: ".ez-relation-content",
+                opacity: ".1"
             });
+    
+            sortable.delegate.after("drag:end",Y.bind(function(e) {
+                this.__setSortedObjectRelationss(e);
+            }, this));
+          },
+          
+          /**
+           * Sets up the new value for the 'destinationContentsIds' and 'relatedContents' attributes when content in the relation list is reordered 
+           * @param {EventFacade} e 
+           */
+          _setSortedObjectRelations: function(e) {
+            var rows = this.get("container").all(".ez-relation-content");
+            var destinationContentsIds = rows._nodes.map(function(row) {
+            return row
+                .getAttribute("data-content-id")
+                .split("/")
+                .pop();
+            });
+
+            var relatedContents = this.get('relatedContents');
+            var sortedRelatedContent = [];
+
+            Y.Array.each(destinationContentsIds, function(sortedContentID){
+                Y.Array.some(relatedContents, function(content){
+                    if(content.get('contentId') == sortedContentID){
+                        sortedRelatedContent.push(content);
+                        return true
+                    }
+                    return false
+                })
+            })
+
+            this.set('relatedContents', sortedRelatedContent);
+            this._set("destinationContentsIds", destinationContentsIds);
```

Comment:
```suggestion
            this._set('destinationContentsIds', destinationContentsIds);
```

Side: Should both use `_set()`?

Reply:
In other places where related content is set `this.set `is used over `this._set` Not sure what the difference is but wanted to be safe 😅

Reply:
ok, me neither so lets stay with this then :)